xbdd-web/src/modules/report/ReportContainer.jsx

```diff
@@ -96,7 +100,30 @@ class ReportContainer extends Component {
     return newExecutions;
   }
 
+  processFailedResponse(response, backupSelectedFeature, backupExecutionHistory) {
+    if (response.status !== 200) {
+      this.setState({
+        selectedFeature: backupSelectedFeature,
+        executionHistory: backupExecutionHistory,
+        isNetworkError: true,
+      });
+      setTimeout(() => {
```

Comment:
Whats this timeout needed for? This is a little weird how this is working.

Reply:
I make a banner to display the error msg and I want it to disappear after 4 seconds. The state will be set false after 4 seconds

Reply:
Cool, we will leave it like this for the moment as discussed but it would be cool to move this to a css transition in the future.